dagshub/data_engine/datasets.py

```diff
@@ -45,6 +46,19 @@ def get_dataset(repo: str, name: Optional[str] = None, id: Optional[Union[int, s
     return sources[-1]
 
 
+def get_dataset_from_file(path: str):
+    """
+    Load a dataset from a local file
+
+    Args:
+        path: Path to the `.dagshub` file with the relevant dataset
```

Comment:
The sphinx syntax requires double backticks for the code blocks

```suggestion
    Args:
        path: Path to the ``.dagshub`` file with the relevant dataset
```